index.html

```diff
@@ -219,12 +219,136 @@ <h2 class="destinations__title">The best destinations</h2>
     <!--SignUp-->
     <section class="sign-up">
       <div class="sign-up__wrap">
-      <img src="./img/user.svg" alt="user" class="sign-up__image" />
-      <p class="sign-up__title">Sign up and get 20% off your first booking</p>
-      <button class="sign-up__btn">Sign up</button>
-      <button class="sign-up__btn-close">
-        <img src="./img/close.svg" alt="X" class="sign-up__image-close" />
-      </button>
+        <img src="./img/user.svg" alt="user" class="sign-up__image" />
+        <p class="sign-up__title">Sign up and get 20% off your first booking</p>
+        <button class="sign-up__btn">Sign up</button>
+        <button class="sign-up__btn-close">
```

Comment:
It's better to use **aria-label** for buttons without text
[Read more](https://developer.mozilla.org/ru/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)